sympy/ntheory/tests/test_primetest.py

```diff
@@ -137,3 +137,9 @@ def test_isprime():
 
 def test_is_square():
     assert [i for i in range(25) if is_square(i)] == [0, 1, 4, 9, 16]
+
+    # issue #17044
+    assert not is_square(216000)
```

Comment:
Looks good.
But I would suggest that it would be better to give a full view of how the numbers are generated from its cube roots like `60*60*60`